buildSrc/src/main/resources/minecraft_specific_words.txt

```diff
@@ -190,6 +190,7 @@ opengl
 checkbox
 hotbar
 hotkeys
+inworld
```

Comment:
no thanks, this should be 2 separate words unless I'm missing something

Reply:
This is needed for some static fields in `net/minecraft/client/gui/screen/Screen`.

Reply:
`Identifier INWORLD_MENU_BACKGROUND = new Identifier("textures/gui/inworld_menu_background.png");`
Maybe it's ok if this differs from the string?

Reply:
What would you suggest it be changed to? I kept it the same as Mojang's mapping, and Yarn's just adds `_TEXTURE` as a suffix.

Reply:
`IN_WORLD_MENU_BACKGROUND`
But I'd like another opinion before deviating from a string.

Reply:
I should have recognised it was a string, I'd rather keep it as inworld instead of deviating